src/MultiCoinNetwork.php

```diff
@@ -51,8 +51,7 @@ protected function genBase58PrefixMap($params) {
         // Prefer scripthash2 to scripthash. For coins like LTC that
         // changed p2sh prefix after-launch to differentiate from BTC.
         // Overrid this method if you need a different behavior.
-        $scripthash = @$prefixes['scripthash2'] ?
```

Comment:
I don't understand why this would generate an error.  The '@' tells php to suppress notices/warnings during evaluation of the array lookup.    Note that using isset() changes the logic slightly because a zero or null value would evaluate to true.  Still, I'm ok with it.

Reply:
I believe that because I have set error_reporting to -1